src/components/Dashboard.tsx

```diff
@@ -0,0 +1,21 @@
+import React, {useEffect}  from 'react';
+import { connect } from 'react-redux';
+import { data } from '../store/store';
+import { getDataThunk } from '../store/actions';
+
+const Dashboard: React.FC = (props: any ) => {
+  useEffect(() => { props. getDataThunk() }, []);
```

Comment:
It's a good start. I think here should be some `setInterval` to poll the api for the current data updates.

Reply:
Done